modules/swagger-codegen/src/main/resources/JavaSpring/apiController.mustache

```diff
@@ -17,8 +17,10 @@ import org.springframework.web.bind.annotation.RequestHeader;
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RequestPart;
 import org.springframework.web.multipart.MultipartFile;
+{{^useSpringCloudClient}}
```

Comment:
Spring cloud client doesn't generate the apiController so I guess this is not needed...